pkg/internal/flags/flags.go

```diff
@@ -26,6 +26,7 @@ import (
 type WatchFlags struct {
 	ReconcilePeriod time.Duration
 	WatchesFile     string
+	MaxWorkers      int
```

Comment:
I think that unless we are adding support for helm in this PR then we need to add this to the ansible specific flags.

Reply:
Done, I moved it to `./pkg/ansible/flags/flag.go`